video/out/wayland_common.c

```diff
@@ -1015,6 +1017,8 @@ static void feedback_presented(void *data, struct wp_presentation_feedback *fbac
 
 static void feedback_discarded(void *data, struct wp_presentation_feedback *fback)
 {
+    if (fback)
+        wp_presentation_feedback_destroy(fback);
```

Comment:
Could you pull these two `wp_presentation_feedback_destroy` calls out into a separate commit?